packages/console/src/components/UserName/index.tsx

```diff
@@ -1,4 +1,4 @@
-import type { User } from '@logto/schemas';
+import type { UserWithRoleNames } from '@logto/schemas';
```

Comment:
`UserWithRoleNames` looks hacky - How about the old fashion `UserDto`?

Reply:
This will be removed after RBAC project is about to complete.